db/fixtures/ae_datastore/ManageIQ/Cloud/Orchestration/Operations/Methods.class/__methods__/available_os_types.rb

```diff
@@ -1,28 +1,46 @@
 #
 # Description: provide the dynamic list content from available operating systems
 #
-os_list = {'unknown' => '<Unknown>', 'linux' => 'Linux', 'windows' => 'Windows'}
-selected_os = 'unknown'
+class AvailableOsTypes
```

Comment:
@pkomanek 
Can we follow Jason's suggestion and create everything with namespaces. The call at the bottom of the file and the specs would have to change.